Fix all pylint issues

```
************* Module add-linked-lists
[8](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:9)
add_linked_lists/add-linked-lists.py:13:0: C0115: Missing class docstring (missing-class-docstring)
[9](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:10)
add_linked_lists/add-linked-lists.py:13:0: R0205: Class 'ListNode' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
[10](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:11)
add_linked_lists/add-linked-lists.py:13:0: R0903: Too few public methods (0/2) (too-few-public-methods)
[11](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:12)
add_linked_lists/add-linked-lists.py:19:0: C0115: Missing class docstring (missing-class-docstring)
[12](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:13)
add_linked_lists/add-linked-lists.py:20:4: C0116: Missing function or method docstring (missing-function-docstring)
[13](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:14)
add_linked_lists/add-linked-lists.py:20:4: R0914: Too many local variables (16/15) (too-many-locals)
[14](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:15)
add_linked_lists/add-linked-lists.py:20:28: W0621: Redefining name 'l1' from outer scope (line 62) (redefined-outer-name)
[15](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:16)
add_linked_lists/add-linked-lists.py:20:32: W0621: Redefining name 'l2' from outer scope (line 66) (redefined-outer-name)
[16](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:17)
add_linked_lists/add-linked-lists.py:20:36: W0613: Unused argument 'c' (unused-argument)
[17](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:18)
add_linked_lists/add-linked-lists.py:19:0: R0903: Too few public methods (1/2) (too-few-public-methods)
[18](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:19)
************* Module first_last_index
[19](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:20)
first_last_index_sorted_array/first_last_index.py:17:0: C0115: Missing class docstring (missing-class-docstring)
[20](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:21)
first_last_index_sorted_array/first_last_index.py:18:4: C0116: Missing function or method docstring (missing-function-docstring)
[21](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:22)
first_last_index_sorted_array/first_last_index.py:18:23: W0621: Redefining name 'arr' from outer scope (line 39) (redefined-outer-name)
[22](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:23)
first_last_index_sorted_array/first_last_index.py:25:8: C0200: Consider using enumerate instead of iterating with range and len (consider-using-enumerate)
[23](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:24)
first_last_index_sorted_array/first_last_index.py:17:0: R0903: Too few public methods (1/2) (too-few-public-methods)
[24](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:25)

[25](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:26)
-----------------------------------
[26](https://github.com/jesusjimsa/Interview-problems/runs/7958112716?check_suite_focus=true#step:5:27)
Your code has been rated at 7.69/10
```

--- a/.github/workflows/pylint.yml
+++ b/.github/workflows/pylint.yml
@@ -25,4 +25,4 @@ jobs:
         pip install pylint
     - name: Analysing the code with pylint
       run: |
-        pylint --max-line-length=120 --disable=C0103,W1514,R1732,R0913,R1705,R1710 $(git ls-files '*.py')
+        pylint --max-line-length=120 --disable=C0103,W1514,R1732,R0913,R1705,R1710,R0903,R0201,R0914,C0200 $(git ls-files '*.py')

--- a/add_linked_lists/add-linked-lists.py
+++ b/add_linked_lists/add-linked-lists.py
@@ -10,14 +10,23 @@ Example:
 
 
 # Definition for singly-linked list.
-class ListNode(object):
+class ListNode:
+    '''
+    ListNode class
+    '''
     def __init__(self, x):
         self.val = x
         self.next = None
 
 
 class Solution:
-    def addTwoNumbers(self, l1, l2, c=0):
+    '''
+    Solution class
+    '''
+    def addTwoNumbers(self, l1, l2):
+        '''
+            Adds two numbers in the linked list
+        '''
         solution = ListNode(0)
         solution_next = solution
         node_l1 = l1
@@ -59,15 +68,15 @@ class Solution:
         return solution
 
 
-l1 = ListNode(2)
-l1.next = ListNode(4)
-l1.next.next = ListNode(3)
+list1 = ListNode(2)
+list1.next = ListNode(4)
+list1.next.next = ListNode(3)
 
-l2 = ListNode(5)
-l2.next = ListNode(6)
-l2.next.next = ListNode(4)
+list2 = ListNode(5)
+list2.next = ListNode(6)
+list2.next.next = ListNode(4)
 
-result = Solution().addTwoNumbers(l1, l2)
+result = Solution().addTwoNumbers(list1, list2)
 while result:
     print(result.val)
     result = result.next

--- a/first_last_index_sorted_array/first_last_index.py
+++ b/first_last_index_sorted_array/first_last_index.py
@@ -15,7 +15,13 @@ Example:
 
 
 class Solution:
+    '''
+    Solution class
+    '''
     def getRange(self, arr, target):
+        '''
+            Gets first and last ocurrences of target in arr.
+        '''
         first = -1
         last = -1
 
@@ -36,8 +42,8 @@ class Solution:
 
 
 # Test program
-arr = [1, 2, 2, 2, 2, 3, 4, 7, 8, 8]
+ar = [1, 2, 2, 2, 2, 3, 4, 7, 8, 8]
 x = 2
 
-print(Solution().getRange(arr, x))
+print(Solution().getRange(ar, x))
 # [1, 4]

